projects/03app_sailbot/03app_sailbot.c

```diff
@@ -109,6 +109,7 @@ static float  calculate_error(float heading, float bearing);
 static int8_t map_error_to_rudder_angle(float error);
 static void   _timeout_check(void);
 static void   _advertise(void);
+static void   _send_gps_data(nmea_gprmc_t *data);
```

Comment:
The data could be made const:

```suggestion
static void   _send_gps_data(const nmea_gprmc_t *data);
```